ballerina/duplicate_field_remover.bal

```diff
@@ -73,7 +73,7 @@ class DuplicateFieldRemover {
         }
     }
 
-    public isolated function visitArgument(parser:ArgumentNode argumentNode, anydata data = ()) {
+    public isolated function visitArgument(parser:InputObjectNode argumentNode, anydata data = ()) {
```

Comment:
In second thoughts, can we rename the `InputObjectNode` to `ArgumentNode`? I feel the correct name would be `ArgumentNode` since we call them arguments in the document, unlike the `InputObject` which is used in the schema.

Reply:
I'll rename the ```InputObjectNode``` to ```ArgumentNode```.